api/v1alpha1/component_types.go

```diff
@@ -32,7 +32,7 @@ type ComponentStatus struct {
 	// The name of the component may come from helm chart name
 	Name string `json:"name"`
 	// versions contains all version of one component.
-	Versions []ComponetVersions `json:"versions"`
+	Versions []ComponentVersions `json:"versions"`
```

Comment:
add `Deprecated` in version
![image](https://github.com/kubebb/core/assets/30621793/546aeddc-20a7-486f-81d4-5d7b2dee0075)